operator/hack/common.sh

```diff
@@ -65,6 +65,26 @@ function retry() {
   return 1
 }
 
+function check_version_tag() {
+  local -r version_tag="$1"
+  local -r allow_dirty_tag="$2"
+
+  echo $IMAGE_TAG_BASE
+  if docker manifest inspect $IMAGE_TAG_BASE:$version_tag ; then
+    return 0
+  else
+    log "Cannot find $IMAGE_TAG_BASE:$version_tag image tag in the registry."
+  fi
+
+
+  if [[ $version_tag == *-dirty ]] && [[ $allow_dirty_tag = false ]]; then
```

Comment:
Please add quotes to variables. 
Also should the if contain `==` equals?
```suggestion
  if [[ "$version_tag" == *-dirty ]] && [[ "$allow_dirty_tag" == false ]]; then
```